include/node_api.h

```diff
@@ -697,6 +697,12 @@ NAPI_EXTERN napi_status napi_get_value_bigint_words(napi_env env,
                                                     int* sign_bit,
                                                     size_t* word_count,
                                                     uint64_t* words);
+NAPI_EXTERN napi_status napi_add_finalizer(napi_env env,
```

Comment:
A new API? We should document this function at our unsupported list if this is not implemented :)